lib/Controller/RoomController.php

```diff
@@ -228,11 +234,41 @@ protected function formatRoom(Room $room, Participant $participant = null) {
 			$room->cleanGuestParticipants();
 		}
 
+		$lastMessageFromHistory = $this->chatManager->getHistory($room->getId(), 0, 1);
+		$lastMessage = '';
```

Comment:
Please default to
```php
$lastMessage = [];
```

That way you don't get a string in one case and an object in the other.